### PR TITLE
publisher argument to force mass monitored items

### PR DESCRIPTION
**Pull request by cristipogacean**

### CODE REVIEW

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Storage/PublishedNodesJobConverter.cs [hunk 1]
```diff
@@ -179,14 +180,38 @@ public class PublishedNodesJobConverter {
         /// Get the node models from entry
         /// </summary>
         /// <param name="item"></param>
+        /// <param name="massMonitoredItems"></param>
         /// <returns></returns>
-        private IEnumerable<OpcNodeModel> GetNodeModels(PublishedNodesEntryModel item) {
+        private IEnumerable<OpcNodeModel> GetNodeModels(PublishedNodesEntryModel item,
+            int massMonitoredItems = 1) {
```
Comment (marcschier) [inline components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Storage/PublishedNodesJobConverter.cs:186]: Maybe "repeatCountForScaleTesting"

File: modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs [hunk 2]
```diff
@@ -82,6 +82,11 @@ public static class LegacyCliConfigKeys {
         /// </summary>
         public const string MaxMessageSize = "MaxMessageSize";
 
+        /// <summary>
+        /// Key for the trusted issuer certificates.
```
Comment (marcschier) [inline modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs:86]: Comment text

File: modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs [hunk 3]
```diff
@@ -82,6 +82,11 @@ public static class LegacyCliConfigKeys {
...
+        /// </summary>
+        public const string MassMonitoredItems = "MassMonitoredItems";
```
Comment (marcschier) [inline modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs:88]: Maybe "ScaleTestCount" or so.

File: modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs [hunk 4]
```diff
@@ -149,6 +149,8 @@ public class LegacyCliOptions : Dictionary<string, string>, IAgentConfigProvider
                         (int i) => this[LegacyCliConfigKeys.BatchSize] = i.ToString() },
                     { "ms|iothubmessagesize=", "the maximum size of the (IoT D2C) message.",
                         (int i) => this[LegacyCliConfigKeys.MaxMessageSize] = i.ToString() },
+                    { "mn|massmonitoreditems=", "number of monitored items clones for load tests",
```
Comment (marcschier) [inline modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs:152]: Yeah, maybe it is a good name :-)

File: modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs (hunk 3 above)
Comment (cristipogacean) [reply modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs:88]: I like ScaleTestCount: chaning everywhere

File: modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs (hunk 4 above)
Comment (cristipogacean) [reply modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliOptions.cs:152]: done

File: modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs (hunk 2 above)
Comment (cristipogacean) [reply modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Runtime/LegacyCliConfigKeys.cs:86]: right

File: components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Storage/PublishedNodesJobConverter.cs (hunk 1 above)
Comment (cristipogacean) [reply components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Storage/PublishedNodesJobConverter.cs:186]: done

**Pull request merged**